### DISCUSSION TITLE
What font used in the logo?

### DISCUSSION THREAD

**Discussion by w3teal** [is-a-dev/register#13830] [Q&A] [answered]
Is the logo was created with an font or custom?
[![banner](https://github.com/is-a-dev/register/assets/143163098/88ee901b-2d27-4a29-acb7-fe67b26a3279)](https://raw.githubusercontent.com/is-a-dev/register/main/media/banner.png)

**Comment by xkcdnerd** [comment]
![image](https://github.com/is-a-dev/register/assets/167662909/7cb4a024-d164-4134-98ab-3cb64321f61f)
closest font is SCR-I 10
I think this isn't a actual font tho

**Reply by w3teal** [reply]
That was an different font, but close

**Reply by xkcdnerd** [reply, marked as answer]
but im saying that the is-a.dev logo font isnt a real font
